provider/lxd/testing_test.go

```diff
@@ -525,6 +525,13 @@ func (conn *StubClient) GetProfile(name string) (*api.Profile, string, error) {
 	return conn.Profile, "etag", conn.NextErr()
 }
 
+func (conn *StubClient) GetContainerProfiles(name string) ([]string, error) {
+	conn.AddCall("GetContainerProfiles", name)
+	return []string{
+		"default",
```

Comment:
by this I'm assuming you mean `juju-${model-name}` and not actually `juju-model-name`. You think it would be ok to just use `juju-model-name` as a constant string?